builds.yml

```diff
@@ -65,6 +65,7 @@ buildTypes:
     # will not be removed
     features:
       - build-flask
+      - institutional-snap
```

Comment:
If this is just a means to exclude the code until it's ready, would it simplify the CI and code, plus give us more stability if we included it all, but just didn't include the preinstalled snap until later? 

And the new controller could just skip the hooks if the snap wasn't found?

Reply:
Have removed all the code fences

We're happy to ship the snap to main whenever now, so there's no need to keep it in flask

The reason for doing it like this was basically that we needed to distribute builds with the snap built in to custodians so they could test (without making them build the snap, and exposing them to all that extra work)